docs/modeling/compound-models.rst

```diff
@@ -100,8 +100,9 @@ Some terminology
 
 It is possible to create new models just by
 combining existing models using the arithmetic operators ``+``, ``-``, ``*``,
-``/``, and ``**``, as well as by model composition using ``|`` and
-concatenation (explained below) with ``&``.
+``/``, and ``**``, or by model composition using ``|`` and
+concatenation (explained below) with ``&``, as well as using the :func:`~astropy.modeling.fix_inputs`
```

Comment:
How is fix_inputs different than setting a parameter to be fixed in a model?  Would be good to clarify the difference.